source/code-of-conduct.rst

```diff
@@ -124,14 +124,14 @@ the community.
 Attribution
 ===========
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+This Code of Conduct is adapted from the [Contributor Covenant] `homepage`_,
```

Comment:
In markdown `[Contributor Covenant][homepage]` only renders `Contributor Covenant` as a link. Please fix accordingly

Reply:
fixed